src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -167,6 +167,7 @@ def setUp(self):
         super(TestWithServers, self).setUp()
 
         self.init_server_param()
```

Comment:
I don't know that this needs it's own method.  It can probably be collapsed down right here.